packages/k8s/tests/k8s-utils-test.ts

```diff
@@ -221,4 +225,32 @@ describe('k8s utils', () => {
       expect(() => containerPorts({ portMappings: ['1/tcp/udp'] })).toThrow()
     })
   })
+
+  describe('generate container name', () => {
+    it('should return the container name from image string', () => {
+      expect(
+        generateContainerName('public.ecr.aws/localstack/localstack')
+      ).toEqual('localstack')
+      expect(
+        generateContainerName(
+          'public.ecr.aws/url/with/multiple/slashes/postgres:latest'
+        )
+      ).toEqual('postgres')
+      expect(generateContainerName('postgres')).toEqual('postgres')
```

Comment:
NIT: Could we use something like a `test.each` here instead of all `expect` in a single unit test?

Reply:
We can of course, but I'd like to counter argue that the tests I've written here are consistent with the rest of the codebase (I couldn't find another `test.each` and [tests a few lines above](https://github.com/actions/runner-container-hooks/blob/bab39cc413ef6751d58f6100694e8737cf45666e/packages/k8s/tests/k8s-utils-test.ts#L205-L216) also have all expect in the unit tests  😉 )

Reply:
We'll look at the tests in a later PR :)